src/providers/data_providers/baseBusinessImageProvider.ts

```diff
@@ -1,11 +1,10 @@
-import { defaultConfig, DataSourceConfiguration } from '../../models/data_models/dataSourceConfiguration';
-import { BusinessInfoInput } from '../../models/data_models/types';
+import { BusinessInfoInput, Geocode } from '../../models/data_models/types';
 import { Image } from '../../models/data_models/image';
 import { BusinessImageProvider } from './businessImageProvider';
+import { DataSourceConfiguration } from '../../models/data_models/dataSourceConfiguration';
 
 export abstract class BaseBusinessImageProvider implements BusinessImageProvider {
-	protected config: DataSourceConfiguration =
-		(process.env.BUSINESS_API_CONFIG && JSON.parse(process.env.BUSINESS_API_CONFIG)) || defaultConfig;
+	constructor(protected config: DataSourceConfiguration) {}
```

Comment:
```suggestion
	constructor(protected readonly config: DataSourceConfiguration) {}
```

nit: since there are other changes to make, we might as well add a bit of const safety here. Your providers share the same config object, so it'd be pretty nice to prevent one provider from messing with the config of another